flags.css

```diff
@@ -1,4 +1,16 @@
 /* Indonesia */
+.rectangle{
+  height: 90px;
+  width: 275px;
+}
+
+.redRectangle{
```

Comment:
while these class names are fine, you are likely to find other things that might be called redRectangle.  Also this has nothing to do with being a rectangle, since it doesn't define the size or shape.  So really it's just red.

See above for how I would probably do it.